components/ImprovedSearch/Hits.tsx

```diff
@@ -28,6 +34,18 @@ type Props = {
 export const Hits = () => {
   const { hits } = useHits()
   const { query } = useSearchBox()
+
+  if (!hits || hits.length === 0) {
+    return (
+      <StyledList variant="numbered">
+        <NoRelevant>
```

Comment:
It's not allowed to add a span inside an ordered list, we need another container than a list to wrap this

Reply:
ok 

> The "Filter" header does still not show up, did you agree with Espen to do that in another PR?

yes i thought to have it in another pat after checking with you once